schema/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ compile_proto_to_cpp(primitive.proto)
 compile_proto_to_cpp(commands.proto)
 compile_proto_to_cpp(queries.proto)
 compile_proto_to_cpp(responses.proto)
+compile_proto_to_cpp(endpoint.proto)
```

Comment:
Endpoint files are already generated in next line, so this line should not be necessary.

Reply:
For some reason there's no `endpoint.pb.h` for me, only grpc one

Reply:
Tried again, worked for some reason, i'll remove it